prime-dotnet-webapi/Services/EmailRenderingService.cs

```diff
@@ -28,17 +32,17 @@ public async Task<Email> RenderBusinessLicenceUploadedEmailAsync(string recipien
                 from: PrimeEmail,
                 to: recipientEmail,
                 subject: "Site Business Licence Uploaded",
-                body: await _razorConverterService.RenderTemplateToStringAsync(RazorTemplates.Emails.BusinessLicenceUploaded, viewModel)
```

Comment:
for several of these, the `body:` named attribute is removed, but some are not. Why is that?

Reply:
That is really weird since i tested these and they were there once
but fixed